Parsers/ASimNetworkSession/Parsers/vimNetworkSessionCiscoISE.yaml

```diff
@@ -0,0 +1,129 @@
+    let GetSrcIpAddr = (src_ip:string) {
+      case ( 
+          src_ip matches regex @"\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}", src_ip, ""
+      )
+    };
+    let GetMacAddr = (mac:string) {
+      case ( 
+          mac matches regex @"[a-fA-F0-9\-:]{17}", mac, ""
+      )
+    };
+    let CiscoISENSParser = (
+        starttime:datetime=datetime(null), 
+        endtime:datetime=datetime(null),
+        srcipaddr_has_any_prefix:dynamic=dynamic([]), 
+        dstipaddr_has_any_prefix:dynamic=dynamic([]), 
+        ipaddr_has_any_prefix:dynamic=dynamic([]),
+        dstportnumber:int=int(null), 
+        hostname_has_any:dynamic=dynamic([]), 
+        dvcaction:dynamic=dynamic([]), 
+        eventresult:string='*', 
+        disabled:bool=false){
+        let src_or_any=set_union(srcipaddr_has_any_prefix, ipaddr_has_any_prefix); 
+        let dst_or_any=set_union(dstipaddr_has_any_prefix, ipaddr_has_any_prefix);
+        let ip_any = set_union(srcipaddr_has_any_prefix, dstipaddr_has_any_prefix, ipaddr_has_any_prefix); 
+        Syslog
+        | where not(disabled)
+        | where ProcessName has_any ("CISE", "CSCO")
+        | parse kind = regex SyslogMessage with @"\d{10}\s" EventOriginalType @"\s(NOTICE|INFO|WARN|WARNING|ERROR|FATAL|DEBUG)"
+        //***************************** <Prefiltering> **************************
+        | where (isnull(starttime) or TimeGenerated>=starttime) 
```

Comment:
Time filters can be applied even before ProcessName filter

Reply:
time filters moved before ProcessName filter